tool/README.md

```diff
@@ -27,7 +27,7 @@ $ mvn clean install
 Issue Tracking
 --------------
 
-Bugs and features are tracked within the WildFly Elytron Tool GitHub project https://github.com/wildfly-security/wildfly-elytron-tool/issues
+Bugs and features are tracked for the WildFly Elytron Tool is through JIRA https://issues.redhat.com/secure/Dashboard.jspa
```

Comment:
Thanks for the PR @suhasid098! 

Just a very minor comment, it would be good to mention the link to the ELY JIRA project specifically. So maybe something like:

Bugs and features for the WildFly Elytron Tool are tracked within the Elytron Jira project at https://issues.jboss.org/browse/ELY